src/quacc/runners/ase.py

```diff
@@ -5,12 +5,16 @@
 import sys
 from importlib.util import find_spec
 from shutil import copy, copytree
-from typing import TYPE_CHECKING, Callable
+from typing import TYPE_CHECKING, Any, Callable
 
 import numpy as np
+from ase import Atoms
+from ase.atoms import Atoms
```

Comment:
Please only use one of these imports. I suggest the latter.

Reply:
Done.